packages/nimble-components/src/rich-text/editor/tests/rich-text-editor-mention.spec.ts

```diff
@@ -1173,14 +1173,16 @@ describe('RichTextEditorMentionListbox', () => {
             expect(pageObject.getSelectedOption()).toBe('user2 name');
         });
 
-        // Intermittent, see https://github.com/ni/nimble/issues/2150
-        xit('should commit mention into the editor when clicked', async () => {
+        it('should commit mention into the editor when clicked', async () => {
             await appendUserMentionConfiguration(element, [
                 { key: 'user:1', displayName: 'username1' }
             ]);
             expect(pageObject.isMentionListboxOpened()).toBeFalse();
             await pageObject.setEditorTextContent('@');
```

Comment:
Yes, we can focus the editor in the 'setEditorTextContent' function. However, we need to use the second approach of conditionally checking if the mention popup is open, then focus and wait. Because the editor is only focused after waiting, Tiptap gets the selection. 

I have updated the code as per the suggestion.

Reply:
lgtm!